tooling/cli/src/helpers/config.rs

```diff
@@ -182,3 +184,26 @@ pub fn get(merge_config: Option<&str>) -> crate::Result<ConfigHandle> {
 pub fn reload(merge_config: Option<&str>) -> crate::Result<ConfigHandle> {
   get_internal(merge_config, true)
 }
+
+pub fn parse_config_from_cli(
+  config: &Option<String>,
+) -> crate::Result<(Option<String>, Option<String>)> {
+  use std::env::{current_dir, set_current_dir};
+
+  let old_cwd = current_dir()?;
+  let cwd = CWD.get().unwrap();
+  set_current_dir(cwd)?;
+
+  let ret = match &config {
+    Some(config) if config.starts_with('{') => (Some(config.to_string()), None),
+    Some(config) => (
+      Some(std::fs::read_to_string(config).with_context(|| "failed to read custom configuration")?),
```

Comment:
instead of the `set_current_dir` thing here we could change `read_to_string(config)` to `read_to_string(cwd.join(config))`. I still feel this solution is kinda weak and could introduce bugs in other places. Maybe we should just duplicate the `set_current_dir` logic to the mobile commands :/

Reply:
sure, reverted the changes and duplicated the logic